### ISSUE TITLE
Brainstorming the different components in the game

### ISSUE THREAD

**Issue by Kevinthesun456** [CSAllenISD/2023-ISP-TowerDefense#14]

**Comment by stefanog1738** [comment]
I agree with what you said, Kevin. I believe that the towers we create need to have their own unique components that make them stand out from others, which would make the game stand out from other tower defense games.

**Comment by SuhraabAlimi** [comment]
I believe that our brainstorming is going through a grand amount of progress, as we have already thought up of multiple towers and their designs, though I thoroughly agree that we need to think up of more towers and the enemies with unique components, as I feel we still have a long ways to go before we can completely express a specific "theme" for the game.
